packages/auth/package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
```

Comment:
The auth service's new responsibility is to, as part of authentication, connect scopes coming from CC to user's `role` field. This is done every time a user needs a JWT token